### DISCUSSION TITLE
Button to automatically cover overspending from "To Budget"

### DISCUSSION THREAD

**Discussion by alehel** [actualbudget/releases#171] [Feature ideas]
When I click on a category where I've overspent, and I have money left over in the "To Budget" category, I would appreciate it if there was an extra button at the bottom of the menu called "Cover overspending from unused budget" or something like that which automatically moves money from the To Budget category to cover the overspent. If there is no money left in the "To Budget" category, this option would be either greyed out or simply not visible. 

![image](https://user-images.githubusercontent.com/22277624/148500153-2dee207d-a607-446b-93c9-3bca7190c253.png)

I know it is relatively easy to use the option "Cover overspending" and then simply search for "To Budget" category however, 99% of the time, I'm covering overspending with available funds. 

Alternatively there could be a button by the "To Budget" category which would automatically move money from the "To Budget" category to cover all categories which are overspent.

**Comment by Deguello** [comment]
I literally never cover overspending from "To Budget" because I keep my "To Budget" at $0.00.

However, what would be super cool in this area, IMO, (though obviously frosting on the frosting on the cake) would be for Actual to remember the last category from which one covered overspending in a category.

For example, I have a "Fun" category which basically serves as a pool for other categories that I want to track but don't want to budget ahead of time, such as "Restaurants," "Socializing," etc. Whenever I categorize to one of these pseudo-child categories, I cover it from "Fun." If Actual could remember that, it would be awesome.

Maybe a more common example would be Christmas spending. A lot of people have a Christmas category with child categories for each person and a pool category that gets funded and drawn from. Having Actual remember that these categories get covered by the Christmas pool category so one automatically gets a "Cover overspending from Christmas Pool" option in the menu would be greatness. Presumably, "To Budget" could be lumped into this such that people who don't keep "To Budget" at $0.00 would get this option.

**Reply by alehel** [reply]
Remembering the last category would achieve the same thing for me, so that would probably be a better solution.

**Comment by bpaulien** [comment]
I'm not sure I'd like a global "all categories which are overspent" but I'd be happy being presented with a checklist of all overspent categories, and be able to select which one(s) I'd want to apply the extra funds to.  But like @Deguello mentioned, I never keep funds in my To Budget category.  I do have a "slush fund" category though, which I regularly cover overspending from.

**Comment by alehel** [comment]
After having used Actual Budget for about 2 weeks now, I think agree with @Deguello . If the autocomplete could show last used category as a suggestion before I start typing and get auto-complete suggestions, that would make the process very smooth.
